### PR TITLE
[Doc] update routine load sql stmt

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md [hunk 1]
```diff
@@ -4,42 +4,123 @@ displayed_sidebar: "English"
...
+| --------------------------------- | ------------ | ------------------------------------------------------------ |
+| db_name                           | No           | The name of the database to which the load job belongs.      |
+| job_name                          | Yes          | The name of the load job.                                    |
+| ALL                               | No           | Displays all load jobs, including those in the `STOPPED` or  `CANCELLED` states. |
+| STATE                             | No           | State of the load job.                                       |
+| ORDER BY field_name [ASC \| DESC] | No           | Sorts the return result in ascending or descending order based on the specified field. The following fields are supported: `Id`, `Name`, `CreateTime`, `PauseTime`, `EndTime`, `TableName`, `State`, and `CurrentTaskNum`.To sort the return result in ascending order, specify `ORDER BY field_name ASC`.To sort the return result in descending order, specify `ORDER BY field_name DESC`.If you do not specify the field or the sort order, the return result is sorted in ascending order of `Id` by default. |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md:38]: image
I guess you may want to make an unordered list here

File: docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md [hunk 2]
```diff
@@ -4,42 +4,123 @@ displayed_sidebar: "English"
...
+| LIMIT limit                       | No           | The number of load jobs that will be returned. If this parameter is not specified, the information of all load jobs that match the filter conditions are displayed. For example, if `LIMIT 10` is specified, only the information of 10 load jobs that match filter conditions will be returned. |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md:39]: image
This sentence should be moved forward preceding "If this parameter is not specified, the information of all load jobs that match the filter conditions are displayed." to make the flow of the parameter description more logical.

File: docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md [hunk 3]
```diff
@@ -4,42 +4,123 @@ displayed_sidebar: "English"
...
+| OFFSET offset                     | No           | The `offset` parameter defines the number of load jobs to be skipped. For example, `OFFSET 5` skips the first five load jobs and returns the rest. The value of the `offset` parameter defaults to `0`. |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md:40]: Remove.

File: docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md [hunk 4]
```diff
@@ -4,42 +4,123 @@ displayed_sidebar: "English"
...
...r RESUME ROUTINE LOAD to create or resume a Routine Load job, the load job first enters the `NEED_SCHEDULE` state.</li><li>`RUNNING`: The load job is running. You can view the consumption progress of the Routine Load job through `Statistic` and `Progress`.</li><li>`PAUSED`: The load job is paused. You can refer to `ReasonOfStateChanged` and `ErrorLogUrls` for troubleshooting. After fixing the error, you can use RESUME ROUTINE LOAD to resume the Routine Load job.</li><li>`CANCELLED`: The load job is cancelled. You can refer to `ReasonOfStateChanged` and `ErrorLogUrls` for troubleshooting. However, after fixing the error, you cannot recover the load job in this state.</li><li>`STOPPED`: Stopped state. You cannot recover the load job in this state.</li><li>`UNSTABLE`: Unstable state. A Routine Load job is set in the `UNSTABLE` state if any task within the Routine Load job lags (namely, the difference between the timestamp of the message being consumed and the current time exceeds this FE parameter [`routine_load_unstable_threshold_second`](../../../administration/FE_configuration.md#routine_load_unstable_threshold_second), and unconsumed messages exist in the data source.)</li></ul> |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md:53]: image
Remove

Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD.md:53]: image
Modify the wording as you did for the other statuses above.

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — DISMISSED** [review]

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**